src/validateData.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable max-len */
+function validateData(text, toCase) {
+  const validCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+  const errorMessages = {
+    emptyTextMessage: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    emptyCaseMessage: '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    invalidCaseMassage: `This case is not supported. Available cases: ${validCases.join(', ')}.`,
+  };
```

Comment:
Create these constants outside the function `validateData` to avoid re-creating every time when you execute the function